ios/browser/api/ads/brave_ads.mm

```diff
@@ -1525,14 +1506,15 @@ - (void)getStatementOfAccounts:(void (^)(NSInteger adsReceived,
                                          double estimatedEarnings,
                                          NSDate* nextPaymentDate))completion {
   if (![self isServiceRunning]) {
-    return;
+    return completion(/*adsReceived=*/0, /*estimatedEarnings=*/0,
+                      /*nextPaymentDate=*/nil);
   }
 
-  ads->GetStatementOfAccounts(
+  adsService->GetStatementOfAccounts(
       base::BindOnce(^(brave_ads::mojom::StatementInfoPtr mojom_statement) {
         if (!mojom_statement) {
-          completion(0, 0, nil);
-          return;
+          return completion(/*adsReceived=*/0, /*estimatedEarnings=*/0,
```

Comment:
nit: some param comments use `camelCase` and others use `snake_cake`. Would be great to be consistent. Thanks

Reply:
Fixed, thanks!